lib/WebpackOptionsValidationError.js

```diff
@@ -155,7 +155,7 @@ class WebpackOptionsValidationError extends WebpackError {
 					case "debug":
 						return (
 							`${baseMessage}\n` +
-							"The 'debug' property was removed in webpack 2.\n" +
+							"The 'debug' property was removed since webpack 2.\n" +
```

Comment:
Here aswell please:

since`webpack >= v2.0.0`

Reply:
Not sure I agree with this particular change. It breaks the grammar of the sentence.

Reply:
So I need to just keep `` in ``?